tools/Resources.py

```diff
@@ -17,54 +15,52 @@ def get_resources(self, target, resourcekind, parentid):
         }
         headers = {
             'Content-Type': "application/json",
-            'Accept': "application/json"
+            'Accept': "application/json",
+            'Authorization': "vRealizeOpsToken " + token
         }
         resources = list()
         disable_warnings(exceptions.InsecureRequestWarning)
         try:
             response = requests.get(url,
-                                    auth=HTTPBasicAuth(username=os.environ['USER'], password=os.environ['PASSWORD']),
                                     params=querystring,
                                     verify=False,
                                     headers=headers)
-            # import json
-            # print(json.dumps(response.json(), sort_keys=True,indent=4))
-            if "resourceList" in response.json():
+            try:
                 for resource in response.json()["resourceList"]:
                     res = dict()
                     res['name'] = resource["resourceKey"]["name"]
                     res['uuid'] = resource["identifier"]
                     resources.append(res)
-            else:
-                raise AttributeError("There is no attribute: resourceList")
+            except AttributeError as ar:
+                print("There is no attribute: resourceList", ar.args)
         except HTTPError as err:
             print("Request failed: ", err.args)
```

Comment:
Is there a specific reason, why you are not raising the error? If it fails it will return an empty list and the actual error will occur later.